### PR TITLE
Add links to references to Q1 Goals and Actions

### PR DESCRIPTION
**Pull request by smellems**
Related to #807 

This is how I suggest we add links without having a long references section at the end.  Does this work?  Anything you would keep and put somewhere?
Maybe a "Why 20%" section?

I also removed empty links (we can add when we know what to link) and fixed the ones where the link was just click [here](#).  For accessibility links should be the title (or close) of the page being linked to

### CODE REVIEW

File: _pages/en/strategy-learning-automating-improving.md [hunk 1]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
+- Participate in "Innovation" at ESDC and Innovation Lab design sessions
 - Explore data science opportunities with ESDC Chief Data Office (CDO)
 - Setup a recurring time to share knowledge with colleagues within your team (small, 5-10 people)
 - Find/become a mentor/coach
-- Join groups and participate in discussions on [GCconnex](#Learn)/[GCcollab](#Learn), as well as external communities
+- Join groups and participate in discussions on GCconnex/GCcollab, as well as external communities
 - Take part in Service Canada staff training
 - Register for training/events offered by ESDC and CSPS
 - Start/attend meetups regarding technologies relevant to your work
 - Learn new tools (Containers, Cloud, Agile, etc)
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: this one (learn new tools) seem a repetition of the above but in more general. E.g. a repetition of:
- Stay up to date on trends and technologies inside and outside the GC
- Experiment with cloud technologies

File: _pages/en/strategy-learning-automating-improving.md [hunk 2]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
 ##### Learn
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Should add:
Reach out to your [Security Champion network](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/SLF%20Forum%20Documents/Security%20Champions%20SLF-EN.pptx) to learn about security

File: _pages/en/strategy-learning-automating-improving.md [hunk 3]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
 
 - Stay up to date on trends and technologies inside and outside the GC
-- Teach yourself basic programming. Examples of instructional material can be found [here](#Learn).
-- Experiment with cloud technologies. Click [here](#Learn) to learn how to create a cloud sandbox environment.
-  - Ex: [Cousera](https://www.coursera.org/), [Udemy](https://www.udemy.com/), or [MOOCs](https://www.mooc-list.com/)
-- Read books, articles or documentation. Examples of content can be found [here](https://sara-sabr.github.io/ITStrategy/references-en.html).
+- Teach yourself basic programming
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Should make it easy for anyone, add examples with free online courses:
(e.g. .Net Core, Java, Python, JavaScript)

File: _pages/en/strategy-learning-automating-improving.md [hunk 4]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
+- Experiment with cloud technologies
+- Take online courses - [edX](https://www.edx.org/), [Cousera](https://www.coursera.org/), [Udemy](https://www.udemy.com/), or [MOOCs](https://www.mooc-list.com/)
+- Read books, articles or documentation - See our [References]({{ site.baseurl }}/references-en.html#Q1)
 - Attend events or conferences
 - Request micro missions with groups throughout ESDC or the Government of Canada at large
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Make it easy for readers by adding hyperlinks
micro-mission: http://esdc.prv/en/service-canada/pob/pob_activities/2016/micromissions.shtml

File: _pages/en/strategy-learning-automating-improving.md [hunk 5]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
-- Attend [IITB showcase](#Learn), [Dev CoP](#Learn), dojos or other learning opportunities within IITB and ESDC
-- Participate in "Innovation" at ESDC and [Innovation Lab](#Learn) design sessions
+- Attend IITB showcase, Dev CoP, dojos or other learning opportunities within IITB and ESDC
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Make it easy for readers by adding hyperlinks:
IITB Showcase: http://dialogue/grp/BU6386699/SitePages/IITB_Showcase.aspx
Dev CoP: https://github.com/esdc-edsc/Welcome/blob/master/Recommendations/DevOps_SDLC.md
Dojos: (will need to contact Mike Murphy's team)

File: _pages/en/strategy-learning-automating-improving.md [hunk 6]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
+- Participate in "Innovation" at ESDC and Innovation Lab design sessions
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Make it easy for readers by adding hyperlinks:
Innovation Lab: http://iservice.prv/eng/innovation_lab/index.shtml

File: _pages/en/strategy-learning-automating-improving.md [hunk 7]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
 - Explore data science opportunities with ESDC Chief Data Office (CDO)
 - Setup a recurring time to share knowledge with colleagues within your team (small, 5-10 people)
 - Find/become a mentor/coach
-- Join groups and participate in discussions on [GCconnex](#Learn)/[GCcollab](#Learn), as well as external communities
+- Join groups and participate in discussions on GCconnex/GCcollab, as well as external communities
 - Take part in Service Canada staff training
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Make it easy for readers. @jaysonmc how did you planned our session? Was there an email or a group you contacted that can be added to the list?

File: _pages/en/strategy-learning-automating-improving.md [hunk 8]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
...
 - Register for training/events offered by ESDC and CSPS
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Make it easy for readers:
ESDC is Saba: https://esdc.sabacloud.com/Saba/Web_spf/CA1PRD0006/local
CSPS is GCCampus: https://idp.csps-efpc.gc.ca/

File: _pages/en/strategy-learning-automating-improving.md [hunk 9]
```diff
@@ -54,68 +55,20 @@ IITB teams should:
 ##### Learn
 
 - Stay up to date on trends and technologies inside and outside the GC
-- Teach yourself basic programming. Examples of instructional material can be found [here](#Learn).
-- Experiment with cloud technologies. Click [here](#Learn) to learn how to create a cloud sandbox environment.
-  - Ex: [Cousera](https://www.coursera.org/), [Udemy](https://www.udemy.com/), or [MOOCs](https://www.mooc-list.com/)
-- Read books, articles or documentation. Examples of content can be found [here](https://sara-sabr.github.io/ITStrategy/references-en.html).
+- Teach yourself basic programming
+- Experiment with cloud technologies
```
Comment (remyb2canada) [inline _pages/en/strategy-learning-automating-improving.md]: Make it easy for readers, suggest adding Play with Docker: https://labs.play-with-docker.com/

**Comment by smellems** [comment]
@remyb2canada I added links and made changes based on your comments.

I think we should merge this and create new PRs to make changes to Goals and Guidance

File: _pages/en/strategy-learning-automating-improving.md (hunk 7 above)
Comment (jaysonmc) [reply _pages/en/strategy-learning-automating-improving.md]: The lunch and learn?

File: _pages/en/strategy-learning-automating-improving.md (hunk 3 above)
Comment (jaysonmc) [reply _pages/en/strategy-learning-automating-improving.md]: Added

**Pull request merged**